app/index.js

```diff
@@ -203,7 +203,7 @@ var initialState = {
   },
   walletLoader: {
     spvConnect: globalCfg.get("spv_connect"),
-    spvInput: false,
+    syncInput: false,
```

Comment:
Need to add the remaining new state vars. I believe they're:

- syncError
- syncAttemptRequest 
- syncCall
- peerCount